components/Admin/AdminButton.tsx

```diff
@@ -127,13 +214,67 @@ function AdminButton({
                 checkStyleOverride={styles.checkIcon}
               />  
             </div>
+            <div className={css(styles.buttonContainer)}>
+              <Button label="Submit" size="xsmall" customButtonStyle={styles.submitButton} onClick={_excludeFromFeed} />
+            </div>
+          </div>
+        </div>
+      );      
+    }
+    if (opt.value === "feature") {
+      return (
+        <div>
+          <div className={css(styles.opt)}>
+            <span className={css(styles.iconWrapper, styles.iconMakeFeature)}>{opt.icon}</span>
+            <span className={css(styles.optLabel)}>{opt.label}</span>
+            <span className={css(styles.settingsBtn)} onClick={(event) => {
+              event.stopPropagation();
+              setIsFeaturedSubmenuOpen(!isFeaturedSubmenuOpen)
+            }}>{icons.cog}</span>
+          </div>
+          <div className={css(styles.menuSettings, isFeaturedSubmenuOpen && styles.featuredMenuSettingsVisible)}>
+            <div className={css(styles.checkboxContainer)}>
+              <CheckBox
```

Comment:
having this many @ts-ignore feels a little off. Let's consider making a new component soon! ( whoever has the bandwidth)

Reply:
yeah, this checkbox component needs updating